app/epa/settings.py

```diff
@@ -37,16 +48,15 @@
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.getenv(
-    "EPA_SECRET_KEY", "v@p9^=@lc3#1u_xtx*^xhrv0l3li1(+8ik^k@g-_bzmexb0$7n"
+SECRET_KEY = env(
+    "EPA_SECRET_KEY", default="v@p9^=@lc3#1u_xtx*^xhrv0l3li1(+8ik^k@g-_bzmexb0$7n"

```

Comment:
Does it make sense to have a fixed secret key?

Reply:
Probably not, I think it is legacy code from the first fork

Reply:
```suggestion
    "EPA_SECRET_KEY"

```

Reply:
default secret key is okay, but I added an entry in README to include `EPA_SECRET_KEY` in .env file